docs/AboutUs.md

```diff
@@ -3,43 +3,64 @@ layout: page
 title: About Us
 ---
 
+<head> <title> About Us </title> </head>
+
+# About Us
+---
```

Comment:
Perhaps, if you add this then we will have double title on the page (?)

Reply:
The <title> will set it as tab name, while the # About Us is the <h1> in the webpage.